backend/server.js

```diff
@@ -0,0 +1,356 @@
+  }
+});
+
+// get bookshelf
+app.get('/storycollection/userid/:userid', async (req, res) => {
+  const { userId, storyCollectionId } = req.params;
+  console.log(userId, storyCollectionId);
+  try {
+    const filteredUser = await User.findById(userId);
+
+    if (filteredUser) {
+      const filteredStoryCollection = await StoryCollection.findById(
+        storyCollectionId
+      );
+      res
+        .status(200)
+        .json({ response: filteredStoryCollection, success: true });
+    } else {
+      res.status(404).json({ response: error, success: false });
+    }
+  } catch (error) {
+    res.status(400).json({ response: error, success: false });
+  }
+});
+
+// PATCH req for adding a story-id to the user's id (the save-my-story-moment-button in the end)
+app.patch(
+  '/user/:userId/storycollection/:storyCollectionId',
+  async (req, res) => {
+    const { userId, storyCollectionId } = req.params;
+
+    try {
+      const filteredUser = await User.findById(userId);
+
+      if (filteredUser) {
+        const filteredStoryCollection = await StoryCollection.findById(
+          storyCollectionId
+        );
+        //pushes the new story in to the user's collection"
+        if (filteredStoryCollection) {
+          const updateUser = await User.findByIdAndUpdate(
+            userId,
+            {
+              $push: {
+                storyCollection: filteredStoryCollection
+              }
+            },
+            { new: true }
+          );
+
+          res.status(200).json({ response: 'User is updated', success: true });
+        } else {
+          res.status(404).json({ response: 'Story not found', success: false });
+        }
+      } else {
+        res.status(404).json({ response: 'User not found', success: false });
+      }
+    } catch (error) {
+      res
+        .status(400)
+        .json({ response: 'Some kind of error, sorry', success: false });
+    }
+  }
+);
+
+app.get('/character', async (req, res) => {
+  try {
+    res.status(200).json({ response: character, success: true });
+  } catch (error) {
+    res.status(400).json({ response: error, success: false });
+  }
+});
+
+app.get('/friend', async (req, res) => {
+  try {
+    res.status(200).json({ response: friend, success: true });
+  } catch (error) {
+    res.status(400).json({ response: error, success: false });
+  }
+});
+
+app.get('/friendname', async (req, res) => {
+  try {
+    res.status(200).json({ response: friendName, success: true });
+  } catch (error) {
+    res.status(400).json({ response: error, success: false });
+  }
+});
+
+app.get('/place', async (req, res) => {
+  try {
+    res.status(200).json({ response: place, success: true });
+  } catch (error) {
+    res.status(400).json({ response: error, success: false });
+  }
+});
+
+app.get('/sound', async (req, res) => {
+  try {
+    res.status(200).json({ response: sound, success: true });
+  } catch (error) {
+    res.status(400).json({ response: error, success: false });
+  }
```

Comment:
Would be great to move all of this data from .json files to database collections